api_tests/lib_sdk/actors/actor.ts

```diff
@@ -131,13 +131,14 @@ export class Actor {
 
         const comitClient: ComitClient = this.getComitClient();
 
+        const defaultExpiries = defaultExpiryTimes();
         const payload = {
             alpha_ledger: alphaLedger,
             beta_ledger: betaLedger,
             alpha_asset: alphaAsset,
             beta_asset: betaAsset,
-            alpha_expiry: defaultExpiryTimes().alpha_expiry,
-            beta_expiry: defaultExpiryTimes().beta_expiry,
+            alpha_expiry: defaultExpiries.alpha_expiry,
+            beta_expiry: defaultExpiries.beta_expiry,
```

Comment:
🙃
Make `defaultExpiryTimes()` return `{ alpha_expiry, beta_expiry }` (if not done already) and use `..defaultExpiries()`.

Reply:
cool, didn't know this was possible (running the tests with this proposal just now)